### ISSUE TITLE
Organization-wide registries configuration in .github/dependabot.yml

### ISSUE THREAD

**Issue by AdamVig** [dependabot/dependabot-core#3360]
To use [Dependabot's support for private registries](https://github.blog/2021-03-15-dependabot-private-dependencies/), it looks like we have to configure [the `registries` option](https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#configuration-options-for-private-registries) in `dependabot.yml` for every repository in our organization.

It would be nice if we could share this configuration across the entire organization by in our `.github` repository. Based on https://github.com/dependabot/dependabot-core/issues/1530#issuecomment-661129172, it seems this was planned:
> When we support private registries in GitHub-native Dependabot, we will support organization-level configuration of private repositories.

I couldn't find any mention of this feature in the GitHub documentation. Is this already supported? If not, is this a feature that the team plans to implement?

**Comment by feelepxyz** [comment]
@AdamVig agreed! We're hoping to support this pattern and built the current implementation with that in mind. It will be some time before we get to it but we're planning to support it.

**Comment by asciimike** [comment]
See https://github.com/dependabot/dependabot-core/issues/3312 for another request for this.

**Comment by AraHaan** [comment]
Wait we can store workflows organization wide inside of a repository named ``.github`` with a workflows subdirectory inside it?

**Comment by kierenj** [comment]
Just checking in to see if/where this might lie in terms of a roadmap. We'll probably add something too all our repositories.. unless this is about to land in the next few weeks/months!

**Comment by jurre** [comment]
It's something that we hope to investigate in next couple of weeks, but I can't make any firm commitments on when it would land. We're very interested in supporting something like this though!

**Comment by emtunc** [comment]
This would be super handy. Just came across a use-case for enabling private registry access across all our repos. Although I could quickly write some python to insert these files into repos, native support for a global dependabot config would be helpful!

**Comment by AraHaan** [comment]
I think it would be cool if there was an UI that shows up in our organization that will do this:

- deletes the .github/dependabot.yml file in all of our repositories (but merge their contents together into a single version in our ``.github`` repository.
- deletes all of the repository specific copies of our github actions workflows that are the same throughout all repositories and move them to the same ``.github`` repository (consolidations) excluding repositories which changes them even slightly. (In those repositories the changed file would override the version of it in the ``.github`` repository, but also act as if the other files in the ``.github`` repository were in the same repository.)
- Moves issue/pr templates/forms to the ``.github`` repository as well (when they are all the same as well) excluding repositories which changes them even slightly. (In those repositories the changed file would override the version of it in the ``.github`` repository, but also act as if the other files in the ``.github`` repository were in the same repository.)

Those are the blockers I face on me using a global ``.github`` repository for one of my organizations that has a ton of repos and allows me to consolidate them and also easily modify them (to where those changes can apply to all repositories in a single go).

**Comment by leifwalsh** [comment]
I'd be interested in this same feature, but with enterprise-wide configuration instead of organization-wide!

**Comment by yrsurya** [comment]
Do we have an ETA on this when can we expect it? super interested in this feature

**Comment by EdouardBavoux** [comment]
This would definitely be interesting for us too, it's a pain to align and maintain dependabot config in every repo.

**Comment by jmullo** [comment]
This is really needed.

**Comment by ldemailly** [comment]
👋

**Comment by JonathanAtCenterEdge** [comment]
Would be incredibly helpful in our organization that's migrating off of renovate

**Comment by AraHaan** [comment]
agreed having a org global dependabot.yml that configures all the possible configurations in case a repository in the org needs it is FAR better than repo local ones as that is then less files that are not directly used for the codebase keeping all the clutter low to none.

**Comment by yrsurya** [comment]
not seeing this feature listed yet in their public roadmap yet - https://github.com/orgs/github/projects/4247

It would be great this can be available in 2025

**Comment by antpili** [comment]
+1 not sure how this isn't a feature since global rules can be configured

**Comment by tylertech-lee-hazlett** [comment]
+1 but i'd like to see it be fine grained and available on the Team level as well as the Org level

**Comment by rogiervanstraten** [comment]
+1

**Comment by carlincherry** [comment]
Org wide registries configuration is now available at the org level - here's the ([changelog](https://github.blog/changelog/2025-07-22-centralized-private-registry-configuration-for-dependabot-is-now-generally-available/) announcement). I'm going to close this as complete, but please give it a reopen if there's something outstanding we missed!
